ethereum/core/src/main/java/org/hyperledger/besu/ethereum/transaction/TransactionSimulator.java

```diff
@@ -80,6 +80,10 @@ public class TransactionSimulator {
               SIGNATURE_ALGORITHM.get().getHalfCurveOrder(),
               (byte) 0);
 
+  // Default signature supplier
+  private static final Supplier<SECPSignature> DEFAULT_SIGNATURE_SUPPLIER =
+      Suppliers.memoize(() -> FAKE_SIGNATURE);
```

Comment:
Could be simplified since the returned value is static

```suggestion
() -> FAKE_SIGNATURE;
```